src/containers/TwinPeaks.jsx

```diff
@@ -0,0 +1,25 @@
+
+import React, { useState } from 'react';
+import Quote from '../components/quote/Quote';
+import Load from '../components/quote/Load';
+import { fetchQuote } from '../services/twinpeaksApi';
+
+const TwinPeaks = () => {
+  const [quote, setQuote] = useState({});
+  
```

Comment:
Probably want a piece of loading state...just incase your fetch to the API fails, it won't blow up your app.